.github/workflows/build-wheels.yml

```
@@ -8,7 +8,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest]
-        python-version: ['3.6']
+        python-version: ['3.8']
```

Should this also include 3.10?
```suggestion
        python-version: ['3.8', '3.10']
```

---

I think one Python platform is enough: it is the base interpreter under which cibuildwheel runs. cibuildwheel itself will download the various runtimes for which it has to build wheels. I chose a slightly older version of Python for stability, but 3.10 might work just as well.